pod.go

```diff
@@ -719,7 +719,7 @@ func (p *Pod) startVM(netNsPath string) error {
 	select {
 	case <-vmStartedCh:
 		break
-	case <-time.After(time.Second):
+	case <-time.After(10 * time.Second):
```

Comment:
This looks fine. Could you add another commit to add a `virtLog.Info("Waiting for %d seconds for VM to start)` call? It might also be worth logging the timeout in the returned error below.

Reply:
Yes I agree that it would be nice to have the timeout error returned somewhere. And also, what do you think about putting an extra debug comment about the time elapsed in case of success, just to make sure we really have some cases taking more time ?
BTW, you should do
```go
case <-time.After(time.Duration(10) * time.Second)
```
since I think `time.After` expect a `time.Duration` type.